src/api.js

```diff
@@ -43,6 +43,10 @@ class Api {
         .use(warp10)
         .use('/api', this.api);
 
+      if (process.env.PROXY_DOCKER_SOCKET) {
+        this.api.use('/container', require('./api/container'));
+      }
+
```

Comment:
We have to decide, for Warp too, if we set env vars we use the middleware or if we always use the middleware and check in it the env var